assets/messages.yml

```diff
@@ -417,12 +398,12 @@ messages:
       shortcut: dup
       category: duplicate
       message: |-
-        *Thank you for your report!*
-        We're tracking this issue in *%s%*, so this ticket is being resolved and linked as a *duplicate*.
+        **Thank you for your report!**
+        We're tracking this issue in **%s%**, so this ticket is being resolved and linked as a **duplicate**.
```

Comment:
These should be sth like https://mojira.atlassian.net/browse/%s%, they will otherwise not become a link in the new jira.

Reply:
Thanks. I have raised this so hopefully the key works in the future but will add this in for the time being